src/libbson/tests/json/bson_corpus/double.json

```diff
@@ -28,16 +28,16 @@
             "relaxed_extjson": "{\"d\" : -1.0001220703125}"
         },
         {
-            "description": "1.23456789012345677E+18",
-            "canonical_bson": "1000000001640081E97DF41022B14300",
-            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"1234567890123456768.0\"}}",
-            "relaxed_extjson": "{\"d\" : 1234567890123456768.0}"
+            "description": "1.2345678921232E+18",
```

Comment:
These changes correspond to https://github.com/mongodb/specifications/commit/2b725f3003b1e444fd15c4738b6e7ecb38e06d8f. We already have [CDRIVER-3500](https://jira.mongodb.org/browse/CDRIVER-3500) open as the downstream ticket, so I've skipped these and referenced that ticket for the time being.